composer.json

```diff
@@ -78,6 +78,9 @@
         "patches": {
             "drupal/field_group": {
                 "https://www.drupal.org/project/field_group/issues/2787179#comment-13467953": "https://www.drupal.org/files/issues/2020-02-17/2787179-highlight-html5-validation-45.patch"
+            },
+            "drupal/inline_entity_form": {
+                "Field name/label disappears when I create a new entity inline": "https://www.drupal.org/files/issues/2020-07-09/inline_entity_form-no_label_required_field_with_no_entries-2842744-23-D8.patch"
```

Comment:
The core team prefers to have a link to the issue (and perhaps comment) instead of the description